web/template/admin/admin_tabs/create-course.gohtml

```diff
@@ -1,61 +1,47 @@
-                                class="w-1/5 btn"
-                                :disabled="tumonlineid===''">
-                            Load Course
-                        </button>
-                    </div>
-                    <div class="px-2" x-show="tumonlineid === '' || numberAttendees !== null">
-                        <div class="bg-gray-50 border rounded-lg dark:bg-secondary-light dark:border-gray-800 px-2 py-1">
-                            <div id="TUMOnlineInfo"
-                                 class="grid gap-1 text-sm text-3 text-center">
-                                <span class="block">
-                                    <i class="fa fa-exclamation-triangle text-warn mr-2"></i>
-                                    <span x-show="tumonlineid === ''">Without a TUMOnline ID you won't be able to make this course public to participants only.</span>
-                                    <span x-show="numberAttendees !== null">Currently there are <span x-text="numberAttendees"></span> students enrolled in this course, please verify that this looks right.</span>
-                                </span>
-                                <a x-show="tumonlineid === ''" target="_blank" class="underline"
-                                   href="https://github.com/joschahenningsen/TUM-Live/wiki/How-do-I-find-the-right-TUMOnline-ID-for-my-course%3F">
-                                    How do I find the right TUMOnline ID for my course?
-                                </a>
-                            </div>
-                        </div>
-                    </div>
-                    <div class="flex space-x-1">
-                        <label for="name" class="hidden">Course Title</label>
-                        <input class="w-4/5 tl-input" id="name" name="name" type="text"
-                               autocomplete="off"
-                               placeholder="Einführung in die Informatik (IN0001)"
-                               x-model="title"
-                               :class="title === '' ? 'border-red-500 focus:border-red-500' : ''"/>
-                        <label for="slug" class="hidden">Slug</label>
-                        <input class="w-1/5 tl-input" id="slug" name="slug" type="text"
-                               autocomplete="off"
-                               x-model="slug"
-                               placeholder="eidi"
-                               :class="slug === '' ? 'border-red-500 focus:border-red-500' : ''"/>
-                    </div>
-                    {{template "semester-selection"}}
-                </form>
-                <br>
-                <button class="btn" @click="admin.createCourse(tumonlineid, semester, year, yearW, title, slug);"
-                        :disabled="slug === '' || title === '' || year === ''">
-                    Create Course
-                </button>
+<div class="form-container grid gap-3"
```

Comment:
I think (`name` and `slug`) and (`semester` and `year`) should be on one line next to each other.

<img width="991" alt="Screenshot 2022-10-17 at 07 06 34" src="https://user-images.githubusercontent.com/29633518/196093766-42b2d441-b436-40f3-8048-0d927d143a0e.png">